.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,25 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Could you please remove these `xml`s'?
You can use `.gitignore` to ignore all files in the .idea directory:
just add the line with `.idea /` to the file